src/MainPage.js

```diff
@@ -31,7 +31,7 @@ const MainPageStyled = styled.div`
     padding: 10px;
     display: grid;
     grid-gap: 20px;
-    background: #f5f5f7;
+    background: var(--silver);
```

Comment:
nice to work with variablen. You might think about it work with main oder primary in the naming. So it's a little bit clearer what kind it is.